k8s/sandbox/cluster-00/jenkins-test/jenkins-pvc.yaml

```diff
@@ -0,0 +1,32 @@
+
+apiVersion: "v1"
+kind: "PersistentVolume"
+metadata:
+  name: "jenkinspv"
+spec:
+  capacity:
+    storage: "20Gi"
+  accessModes:
+    - "ReadWriteOnce"
+  storageClassName: managed-premium
+  persistentVolumeReclaimPolicy: Retain
+  azureDisk:
+    kind: Managed
+    diskName: test-disk
+    diskURI: /subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/aks-infra-sbox-rg/providers/Microsoft.Compute/disks/test-disk
+    cachingMode: ReadWrite 
+    fsType: ext4 
+    readOnly: false
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: jenkins-disk-claim
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: managed-premium
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
probably too small in general, but big enough for a test, any reason you aren't matching your request to the size of the disk you are creating?

Reply:
wrong copy/paste. fixed it